fares-data-build-tool/fdbt-site
# Feature/tfn 346 select expiry multiple products

# Description

This PR introduces a new page upon which the user can select the validity of multiple products at once.

# Testing instructions

-    Pull down this branch and run 'make' in the fdbt-dev repo.
-    Run through the two available user journeys to get to the multipleProductValidity page (i) via the csvZoneUpload page and (ii) via the singleOperator page.
-    Check that the page correctly errors when the user doesn't select a radio button on the page.
-    Upon reaching the thankyou page, run 'awslocal s3api list-objects --bucket fdbt-matching-data-dev' and search for the UUID in the objects in your local s3 bucket.

# Type of change


# Checklist:


[multipleProductValidity.pdf](https://github.com/fares-data-build-tool/fdbt-site/files/4550809/multipleProductValidity.pdf)
